config.js

```diff
@@ -240,7 +241,9 @@ exports.updateFromArgv = function() {
                 + `\nDebug status: ${debugStatus}`);
     // B - test unsupported argv //
     testDroppedArgv(argv);
-    ensureSupportedOgspvAI(argv.ogspv, ogsPvAIs);
+    if (argv.ogspv) {
```

Comment:
Why is this required? Does this if fix bug where `ensureSupportedOgspvAI` cannot handel `argv.ogspv==null`? If so, i would prefer to fix it inside the `ensureSupportedOgspvAI` function.

Reply:
const upperCaseOgsPv = ogspv.toUpperCase();
undefined.toUpperCase is not defined
I had fixed it in there, and was then told to move it here.